stream-parent/stream-client/src/main/java/org/interledger/stream/sender/SimpleStreamSender.java

```diff
@@ -120,33 +146,52 @@ public SimpleStreamSender(final StreamEncryptionService streamEncryptionService,
   public SimpleStreamSender(
       final StreamEncryptionService streamEncryptionService, final Link link, ExecutorService executorService
   ) {
-    this(streamEncryptionService, link, executorService, new StreamConnectionManager());
+    this(streamEncryptionService, link, executorService, new StreamConnectionManager(), Optional.empty());
   }
 
   /**
    * Required-args Constructor.
    *
-   * @param streamEncryptionService A {@link StreamEncryptionService} used to encrypt and decrypted end-to-end STREAM
+   * @param streamEncryptionService An instance of {@link StreamEncryptionService} used to encrypt and decrypted
+   *                                end-to-end STREAM packet data (i.e., packets that should only be visible between
+   *                                sender and receiver).
+   * @param link                    A {@link Link} that is used to send ILPv4 packets to an immediate peer.
+   * @param executorService         executorService to run the payments
+   * @param sendPacketSleep amount of time for a thread to sleep before sending more packets
```

Comment:
Minor nit: formatting

```suggestion
   * @param         sendPacketSleep amount of time for a thread to sleep before sending more packets
```